### PR TITLE
Adding French Translation

### PR DESCRIPTION
**Pull request by 5axes**
First Full French Translation of the Guide.  The translation is complet but thurther modifications will be added ( Check Links, Translate also some image with English text and check Variable NAme)

### CODE REVIEW

File: resources/translations/fr_FR/experimental/small_feature_speed_factor.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Vitesse de petite structure
+===
+
+Les contours plus courts que la [longueur maximale des petits éléments](./small_feature_max_length.md) peuvent être imprimés à une vitesse réduite. Ce paramètre vous permet d'indiquer la vitesse à laquelle ces contours doivent être imprimés, en tant que facteur de leur [vitesse d'impression normale](../vitesse/speed_wall.md).
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/experimental/small_feature_speed_factor.md]: This link is broken.
```suggestion
Les contours plus courts que la [longueur maximale des petits éléments](./small_feature_max_length.md) peuvent être imprimés à une vitesse réduite. Ce paramètre vous permet d'indiquer la vitesse à laquelle ces contours doivent être imprimés, en tant que facteur de leur [vitesse d'impression normale](../speed/speed_wall.md).
```

File: resources/translations/fr_FR/experimental/wipe_hop_speed.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Vitesse du décalage d'essuyage
+===
+
+Ce réglage permet de configurer la vitesse à laquelle la buse est déplacée vers le haut - ou la plaque de montage est déplacée vers le bas - lors de l'exécution du Z hop pour la procédure d'essuyage. Il peut être configuré séparément de la [vitesse du saut Z](../vitesse/speed_z_hop.md) ordinaire.
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/experimental/wipe_hop_speed.md]: This link is broken.
```suggestion
Ce réglage permet de configurer la vitesse à laquelle la buse est déplacée vers le haut - ou la plaque de montage est déplacée vers le bas - lors de l'exécution du Z hop pour la procédure d'essuyage. Il peut être configuré séparément de la [vitesse du saut Z](../speed/speed_z_hop.md) ordinaire.
```

File: resources/translations/fr_FR/experimental/wipe_pause.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+La procédure d'essuyage est suffisamment longue pour un mouvement en mouvement. La buse se déplace sur le côté du volume du bâtiment et effectue un mouvement de va-et-vient, puis doit retourner à l'impression. Pendant ce temps, il y aura beaucoup de suintement et une [rétraction plus longue](./wipe_retraction_amount.md) peut être nécessaire. Après la rétraction, il y aura une période de temps après laquelle l'écoulement de la buse devra toujours commencer. Cette pause permet à la chambre de buse de revenir à la pression lorsque le matériau n'est pas rétracté à nouveau, permettant à la première ligne d'être imprimée correctement sans sous-extrusion importante.
+
+Une pause trop longue provoquera l'apparition d'une goutte à l'emplacement de la buse. En fonction de la [commande d'impression](../infill/infill_before_walls.m), cela peut ne pas être significatif car le point peut apparaître à l'intérieur de l'impression où il n'est pas visible. Cependant, le matériau utilisé pour ce point ne sera pas utilisé pour les lignes imprimées suivantes, ce qui entraînerait une sous-extrusion à cet endroit. Ce paramètre nécessite un réglage soigneux.
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/experimental/wipe_pause.md]: This link is broken.
```suggestion
Une pause trop longue provoquera l'apparition d'une goutte à l'emplacement de la buse. En fonction de la [commande d'impression](../infill/infill_before_walls.md), cela peut ne pas être significatif car le point peut apparaître à l'intérieur de l'impression où il n'est pas visible. Cependant, le matériau utilisé pour ce point ne sera pas utilisé pour les lignes imprimées suivantes, ce qui entraînerait une sous-extrusion à cet endroit. Ce paramètre nécessite un réglage soigneux.
```

File: resources/translations/fr_FR/material/infill_material_flow.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Débit de remplissage
+===
+
+Ce réglage ajuste le débit pour le remplissage uniquement. Le débit de la recharge peut être réglé séparément du débit du reste de l'impression.
+
+L'ajustement du débit pendant le remplissage est une méthode d'arrêt de l'espace pour régler les problèmes de vitesse d'extrusion ou de résistance. Le même effet peut être obtenu en ajustant la [distance entre les lignes](../remplissage/remplissage_distance_ligne.md) et la [largeur de ligne](../résolution/remplissage_ligne_largeur.md) du remplissage, mais ce réglage peut être plus intuitif.
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/material/infill_material_flow.md]: These links are broken.
```suggestion
L'ajustement du débit pendant le remplissage est une méthode d'arrêt de l'espace pour régler les problèmes de vitesse d'extrusion ou de résistance. Le même effet peut être obtenu en ajustant la [distance entre les lignes](../infill/infill_line_distance.md) et la [largeur de ligne](../resolution/infill_line_width.md) du remplissage, mais ce réglage peut être plus intuitif.
```

File: resources/translations/fr_FR/material/infill_material_flow.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+
+Les problèmes de vitesse d'extrusion ou de résistance du remplissage sont généralement dus à l'une des deux causes suivantes Des croisements dans le modèle de remplissage, ou un trop grand changement de débit entre le remplissage et d'autres structures. Plutôt que d'ajuster ce débit, il peut être plus efficace d'ajuster le [schéma de remplissage](../remplissage/plan_remplissage.md) ou la [largeur_ligne](../résolution/largeur_ligne_remplissage.md). Choisissez un motif de remplissage qui ne se croise pas, comme le zigzag, et choisissez une largeur de ligne plus proche du taux d'extrusion des parois et de la peau. Si la largeur de la ligne doit être augmentée pour des raisons de résistance mais que le débit est limité, il est conseillé d'utiliser le [multiplicateur de remplissage](../infill/infill_multiplier.md) plutôt que d'augmenter le débit.
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/material/infill_material_flow.md]: These links are broken.
```suggestion
Les problèmes de vitesse d'extrusion ou de résistance du remplissage sont généralement dus à l'une des deux causes suivantes Des croisements dans le modèle de remplissage, ou un trop grand changement de débit entre le remplissage et d'autres structures. Plutôt que d'ajuster ce débit, il peut être plus efficace d'ajuster le [schéma de remplissage](../infill/infill_pattern.md) ou la [largeur_ligne](../resolution/infill_line_width.md). Choisissez un motif de remplissage qui ne se croise pas, comme le zigzag, et choisissez une largeur de ligne plus proche du taux d'extrusion des parois et de la peau. Si la largeur de la ligne doit être augmentée pour des raisons de résistance mais que le débit est limité, il est conseillé d'utiliser le [multiplicateur de remplissage](../infill/infill_multiplier.md) plutôt que d'augmenter le débit.
```

File: resources/translations/fr_FR/material/support_bottom_material_flow.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+
+En cas de problème de vitesse d'extrusion, il est préférable d'examiner les [vitesses d'impression](../speed/speed_support_bottom.md), [température](material_print_temperature.md) et [largeur de ligne](../resolution/support_bottom_line_width.md). Il y a peut-être une trop grande différence entre le débit du plancher de support et les autres structures de la couche. Peut-être que la largeur de la ligne est trop fine pour être extrudée correctement. Si l'interface est imprimée avec un [matériau différent](../support/support_interface_extruder_nr.md), un problème courant est que le matériau avec lequel l'interface est imprimée n'a pas assez de temps pour commencer à couler correctement. Ce problème peut être résolu en utilisant une [tour principale](../dual/prime_tower_enable.md) ou en augmentant la [surface du plancher de support](../support/support_bottom_offset.md).
+
+Si le support colle trop bien au modèle, le réglage de la [largeur de ligne](../resolution/support_bottom_line width.md) sera généralement plus efficace car il permet également de rapprocher les lignes pour obtenir la même densité de remplissage du support.
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/material/support_bottom_material_flow.md]: This link is broken.
```suggestion
Si le support colle trop bien au modèle, le réglage de la [largeur de ligne](../resolution/support_bottom_line_width.md) sera généralement plus efficace car il permet également de rapprocher les lignes pour obtenir la même densité de remplissage du support.
```

File: resources/translations/fr_FR/material/support_material_flow.md [hunk 7]
```diff
@@ -0,0 +1,8 @@
+Débit du support
+===
+
+Ce réglage permet d'ajuster le débit pour le soutien uniquement. Le débit du support peut être réglé séparément du débit du reste de l'impression.
+
+Le réglage du débit pendant le support est une méthode d'arrêt de l'espace pour régler les problèmes de débit d'extrusion ou d'adhérence entre le support et l'impression. Le même effet peut être obtenu en ajustant la [largeur de ligne](../resolution/support_line_width.md) ou [interligne](../support_ligne_distance.md) du support, mais l'ajustement du débit peut être plus intuitif.
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/material/support_material_flow.md]: This link is broken.
```suggestion
Le réglage du débit pendant le support est une méthode d'arrêt de l'espace pour régler les problèmes de débit d'extrusion ou d'adhérence entre le support et l'impression. Le même effet peut être obtenu en ajustant la [largeur de ligne](../resolution/support_line_width.md) ou [interligne](../support_line_distance.md) du support, mais l'ajustement du débit peut être plus intuitif.
```

File: resources/translations/fr_FR/speed/speed_support_bottom.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
+Vitesse d'impression des bas de support
+===
+
+Ce paramètre configure la vitesse d'impression de la face inférieure du support, si [Activer les bas de support](../support/support_bottom_enable.md).
+
+![Diverses structures imprimées à des vitesses différentes](../../../articles/images/speed_difference.png)
+
+Une vitesse plus élevée peut permettre de gagner du temps d'impression.
+
+L'effet de ce réglage dépend beaucoup des circonstances. Avec certains matériaux, en particulier les matériaux qui refroidissent rapidement et avec une grande [Distance fond support](../support/support_fond_distance.md), une vitesse plus élevée augmentera l'adhérence entre le support et le modèle. L'effet est similaire à celui d'un pontage : A plus grande vitesse, le matériau est moins soigneusement suspendu et n'a pas le temps d'être refroidi par les ventilateurs de la tête d'impression. C'est quelque chose que vous ne voulez pas normalement, car le support laissera une plus grande marque sur votre surface.
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/speed/speed_support_bottom.md]: This link is broken.
```suggestion
L'effet de ce réglage dépend beaucoup des circonstances. Avec certains matériaux, en particulier les matériaux qui refroidissent rapidement et avec une grande [Distance fond support](../support/support_bottom_distance.md), une vitesse plus élevée augmentera l'adhérence entre le support et le modèle. L'effet est similaire à celui d'un pontage : A plus grande vitesse, le matériau est moins soigneusement suspendu et n'a pas le temps d'être refroidi par les ventilateurs de la tête d'impression. C'est quelque chose que vous ne voulez pas normalement, car le support laissera une plus grande marque sur votre surface.
```

File: resources/translations/fr_FR/troubleshooting/bed_adhesion_problems.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+
+De même, si un radeau est utilisé, vous pouvez augmenter l'adhérence en augmentant la [largeur du radeau](../platform_adhesion/raft_margin.md). Le radeau sera toujours imprimé en dessous de tout, car les couches doivent correspondre. Si l'adhérence entre la maquette et le radeau pose problème, pensez à réduire l'[entrefer](../platform_adhesion/raft_airgap.md) entre le radeau et la maquette. De plus, vous pouvez envisager d'augmenter la [largeur de la ligne de base du radeau](../platform_adhesion/raft_base_line_width.md).
+
+Prévention du gauchissement
+----
+Si votre impression souffre de déformation, la zone de contact entre l'impression et la plaque de montage est fortement réduite. Pour éviter cela, consultez l'article sur [le décollement](./warping.md). En résumé, voici quelques ajustements à apporter pour éviter le gauchissement.
+* Réglez le [Modèle de couche initiale du bas](../resolution/layer_height_0.md) sur concentrique.
+* Réduisez la [température de la plaque de construction] (../matériau/matériau_lit_température.md) et augmentez la [température du volume de construction] (../matériau/volume_de_construction_température.md).
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/troubleshooting/bed_adhesion_problems.md]: These links are broken.
```suggestion
* Réduisez la [température de la plaque de construction](../material/material_bed_temperature.md) et augmentez la [température du volume de construction](../material/build_volume_temperature.md).
```

File: resources/translations/fr_FR/troubleshooting/bed_adhesion_problems.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+* Ajustez le modèle de manière à ce qu'il y ait moins de coins extérieurs tranchants sur la première couche.
+
+Ajustements divers
+----
+Ces paramètres peuvent également être ajustés si les méthodes normales d'adhésion au lit ne sont pas suffisantes.
+* Augmenter la [Hauteur de la couche initiale](../résolution/ hauteur_de_la_couche_0.md). Cela offre un peu plus de marge d'erreur, si la plaque de construction n'est pas parfaitement de niveau. Cela augmente également la force avec laquelle le matériau est poussé hors de la buse et sur la plaque de montage.
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/troubleshooting/bed_adhesion_problems.md]: This link is broken.
```suggestion
* Augmenter la [Hauteur de la couche initiale](../resolution/layer_height_0.md). Cela offre un peu plus de marge d'erreur, si la plaque de construction n'est pas parfaitement de niveau. Cela augmente également la force avec laquelle le matériau est poussé hors de la buse et sur la plaque de montage.
```

File: resources/translations/fr_FR/troubleshooting/bed_adhesion_problems.md [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+* Augmenter la [Largeur de la ligne de la couche initiale](../resolution/initial_layer_line_width_factor.md). De même, cela pousse le matériau avec plus de force, en le collant bien à la plaque de montage.
+* L'augmentation de la [couche initiale de la température d'impression] (../material/material_print_temperature_layer_0.md) permet au matériau de s'écouler davantage sur la plaque de montage, ce qui augmente la surface de contact.
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/troubleshooting/bed_adhesion_problems.md]: Broken Markdown syntax.
```suggestion
* L'augmentation de la [couche initiale de la température d'impression](../material/material_print_temperature_layer_0.md) permet au matériau de s'écouler davantage sur la plaque de montage, ce qui augmente la surface de contact.
```

File: resources/translations/fr_FR/troubleshooting/bed_adhesion_problems.md [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+* Comme pour l'augmentation de la température, vous pouvez également réduire la [vitesse du ventilateur de la couche initiale](../cooling/cool_fan_speed_0.md) pour garder le matériau fluide plus longtemps.
+* De même, réduisez la [vitesse de la couche initiale] (../speed/speed_print_layer_0.md). La réduction de la [vitesse d'impression de la couche initiale] (../speed/speed_print_layer_0.md) permet au matériau de s'écouler à nouveau davantage, car le matériau est maintenu fluide plus longtemps par la buse chaude. La réduction de la [vitesse de déplacement de la couche initiale] (../speed/speed_travel_layer_0.md) réduit le risque qu'un mouvement de déplacement arrache les lignes précédemment imprimées de la plaque de montage. De même, il faut envisager d'ajuster les paramètres [Accélération de la couche initiale](../speed/acceleration_layer_0.md) et [Accélération de la couche initiale](../speed/jerk_layer_0.md).
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/troubleshooting/bed_adhesion_problems.md]: Broken Markdown syntax.
```suggestion
* De même, réduisez la [vitesse de la couche initiale](../speed/speed_print_layer_0.md). La réduction de la [vitesse d'impression de la couche initiale](../speed/speed_print_layer_0.md) permet au matériau de s'écouler à nouveau davantage, car le matériau est maintenu fluide plus longtemps par la buse chaude. La réduction de la [vitesse de déplacement de la couche initiale](../speed/speed_travel_layer_0.md) réduit le risque qu'un mouvement de déplacement arrache les lignes précédemment imprimées de la plaque de montage. De même, il faut envisager d'ajuster les paramètres [Accélération de la couche initiale](../speed/acceleration_layer_0.md) et [Accélération de la couche initiale](../speed/jerk_layer_0.md). 
```

File: resources/translations/fr_FR/troubleshooting/sagging.md [hunk 13]
```diff
@@ -0,0 +1,57 @@
...
+* Le réglage de l'[angle de surplomb](../support/support_angle.md) est la méthode la plus simple pour régler la quantité de support à placer. Vous devez régler ce paramètre de manière à ce que toutes vos zones en surplomb soient prises en charge. Ce paramètre détermine également les zones qui seront dessinées en rouge lors de la phase de préparation. La réduction de l'angle de surplomb entraînera l'impression d'un plus grand nombre de supports, ce qui réduira l'affaissement d'un plus grand nombre de parties de l'impression, mais prendra également plus de temps et de matériel et causera plus de cicatrices.
+* Vous pouvez supprimer les parties en surplomb au-dessus des autres parties en réglant le paramètre [Support Placement](../support/support_type.md) sur "Toucher la plaque de montage". Si vous réglez plutôt ce paramètre sur "Partout", vous pouvez éviter les mauvais débordements.
+* Assurez-vous que la [Zone de support minimale](../support/zone_de_support_minimale.md) n'efface pas de minces morceaux de support.
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/troubleshooting/sagging.md]: This link is broken.
```suggestion
* Assurez-vous que la [Zone de support minimale](../support/minimum_support_area.md) n'efface pas de minces morceaux de support.
```

File: resources/translations/fr_FR/experimental/spaghetti_infill_enabled.md [hunk 14]
```diff
@@ -0,0 +1,26 @@
...
+
+L'autre application du remplissage spaghetti est le moulage. Si vous avez une imprimante qui comprend un outil pour couler à des endroits spécifiques, cette fonction peut être utilisée pour remplir votre modèle de manière assez uniforme avec des dépôts de matériaux contrôlés. Si vous utilisez cette fonction, la [densité d'écoulement](./spaghetti_flow.md) doit probablement être d'environ 100 %, en fonction du ratio de contraction/dilatation du matériau coulé.
+
+Réglage d'autres paramètres pour le remplissage de spaghetti
+----
+Ces deux demandes nécessitent un traitement très particulier de votre demande de remplissage. Voici quelques paramètres que vous voudrez très probablement adapter à vos besoins :
+* La [vitesse d'impression du remplissage](../speed/speed_infill.md) doit être massivement réduite. Si la buse doit extruder le matériau de remplissage de 10 couches en même temps, vous devrez réduire la vitesse d'impression d'au moins un facteur 8.
+* Réglez votre [épaisseur de la couche de remplissage] (../infill/infill_sparse_thickness.md) pour qu'elle soit égale à la hauteur de la couche.
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/experimental/spaghetti_infill_enabled.md]: Broken Markdown syntax.
```suggestion
* Réglez votre [épaisseur de la couche de remplissage](../infill/infill_sparse_thickness.md) pour qu'elle soit égale à la hauteur de la couche.
```

File: resources/translations/fr_FR/meshfix/meshfix_maximum_deviation.md [hunk 15]
```diff
@@ -0,0 +1,17 @@
...
+
+L'imprimante doit traiter le g-code pendant qu'il est exécuté. Si le g-code contient de nombreux segments de lignes minuscules, la tête d'impression peut se précipiter dans le mouvement si rapidement que le processeur de l'imprimante 3D ne peut pas suivre. Cela a pour effet que la tête d'impression ralentit sporadiquement pour permettre à l'unité centrale de rattraper le mouvement ou simplement d'attendre la prochaine commande de mouvement. La surface devient alors très rugueuse, voire présente de petites taches, car le débit sortant de la buse ne correspond pas parfaitement au mouvement sporadique de la buse. Parfois, une résolution plus faible se traduit par une meilleure qualité d'impression.
+
+Ce réglage indique dans quelle mesure la nouvelle trajectoire à résolution réduite peut s'écarter de la trajectoire à haute résolution originale. On considère que les lignes sont jointes à d'autres segments de ligne si elles sont plus courtes que la [résolution maximale] (meshfix_maximum_resolution.md), mais si ce raccourci entraîne une déviation du trajet supérieure à la distance indiquée par ce paramètre, les lignes ne seront pas jointes.
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/meshfix/meshfix_maximum_deviation.md]: Broken Markdown syntax.
```suggestion
Ce réglage indique dans quelle mesure la nouvelle trajectoire à résolution réduite peut s'écarter de la trajectoire à haute résolution originale. On considère que les lignes sont jointes à d'autres segments de ligne si elles sont plus courtes que la [résolution maximale](meshfix_maximum_resolution.md), mais si ce raccourci entraîne une déviation du trajet supérieure à la distance indiquée par ce paramètre, les lignes ne seront pas jointes.
```

File: resources/translations/fr_FR/troubleshooting/underextrusion.md [hunk 16]
```diff
@@ -0,0 +1,71 @@
...
+
+Certains microprogrammes sont capables de compenser cet effet. Marlin possède une fonction appelée [Linear Advance] (http://marlinfw.org/docs/features/lin_advance.html) qui commence à augmenter la pression à l'intérieur de la chambre de la buse à l'avance afin qu'elle puisse sortir de la buse plus tôt. Sailfish a une [fonction similaire] (https://www.sailfishfirmware.com/doc/tuning-jkn-advance.html) qu'il appelle JKN Advance. Pour régler les paramètres de ces fonctions, regardez quel g-code vous devez écrire dans le g-code de départ.
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/troubleshooting/underextrusion.md]: Broken Markdown syntax.
```suggestion
Certains microprogrammes sont capables de compenser cet effet. Marlin possède une fonction appelée [Linear Advance](http://marlinfw.org/docs/features/lin_advance.html) qui commence à augmenter la pression à l'intérieur de la chambre de la buse à l'avance afin qu'elle puisse sortir de la buse plus tôt. Sailfish a une [fonction similaire](https://www.sailfishfirmware.com/doc/tuning-jkn-advance.html) qu'il appelle JKN Advance. Pour régler les paramètres de ces fonctions, regardez quel g-code vous devez écrire dans le g-code de départ.
```

File: resources/translations/fr_FR/troubleshooting/warping.md [hunk 17]
```diff
@@ -0,0 +1,36 @@
...
+* Pour réduire les gradients de température à l'intérieur de l'impression, réduisez la [température de la plaque de construction](../material/material_bed_temperature.md). L'objet devrait alors refroidir de manière plus uniforme.
+* Selon le matériel de votre imprimante, l'impression dans une chambre chauffée fermée peut également empêcher le gauchissement. D'une part, cela réduira la différence de température entre le volume de construction et la plaque de construction, et la différence entre le volume de construction et la buse. Cela permettra également au plastique de [recuire] (https://en.wikipedia.org/wiki/Annealing_%28glass%29) plus longtemps, réduisant ainsi les contraintes internes et donc la traction des lignes imprimées qui causent le gauchissement. Si votre imprimante le permet, pensez à augmenter la [température du volume de construction](../material/build_volume_temperature.md).
```
Comment (Ghostkeeper) [inline resources/translations/fr_FR/troubleshooting/warping.md]: Broken Markdown syntax.
```suggestion
* Selon le matériel de votre imprimante, l'impression dans une chambre chauffée fermée peut également empêcher le gauchissement. D'une part, cela réduira la différence de température entre le volume de construction et la plaque de construction, et la différence entre le volume de construction et la buse. Cela permettra également au plastique de [recuire](https://en.wikipedia.org/wiki/Annealing_%28glass%29) plus longtemps, réduisant ainsi les contraintes internes et donc la traction des lignes imprimées qui causent le gauchissement. Si votre imprimante le permet, pensez à augmenter la [température du volume de construction](../material/build_volume_temperature.md).
```

File: resources/articles/experimental/support_tree_branch_distance.md [hunk 18]
```diff
@@ -11,4 +11,4 @@ By reducing the branch distance, a better overhang quality can be achieved becau
 
 However reducing the branch distance will also cause the support to take more material and printing time.
 
-Reducing the branch distance below the [branch diameter](support_tree_branch_diameter.md) will cause the branches to merge before they could properly be formed. As a result the centre of large overhang areas may not get supported properly then.
\ No newline at end of file
+Reducing the branch distance below the [Diamètre des branches de support arborescent](./support_tree_branch_diameter.md) will cause the branches to merge before they could properly be formed. As a result the centre of large overhang areas may not get supported properly then.
```
Comment (Ghostkeeper) [inline resources/articles/experimental/support_tree_branch_distance.md]: I don't think you intended to edit the English article here.
```suggestion
Reducing the branch distance below the [branch diameter](support_tree_branch_diameter.md) will cause the branches to merge before they could properly be formed. As a result the centre of large overhang areas may not get supported properly then.
```

Comment (5axes) [reply resources/articles/experimental/support_tree_branch_distance.md]: > I don't think you intended to edit the English article here.

Yes it was certainly a mistake

**Comment by Ghostkeeper** [comment]
I've reviewed the whole thing (glossing over the texts rather quickly). I've found a few consistency mistakes that I'll apply myself after merging. Of course, with a text on this scale, we can't expect to reach 100% consistency, especially when the original text is also moving. This was one of the reasons why I gave the button for switching translations a pretty prominent spot.

One thing you may not have seen is that the translations for "Enable Print Cooling" and "Raft Fan Speed" are still missing.

**Comment by Ghostkeeper** [comment]
I don't think I've ever merged 1339 commits in one go. Too bad, they were actually 2 commits too many!

**Comment by 5axes** [comment]
Thank's a lot Ghostkeeper.
I've notice that we must use now four equal signs for article headers . 

Can we continue to review the translation ( change some texte , or add some explanations ) or do you prefer for the moment to have a break to focus your time on other contribution) .  One our side I will try to continue to get some feedback from the french user , and include this modification in my  own Fork.

File: resources/translations/fr_FR/about/preferences.md [hunk 19]
```diff
@@ -0,0 +1,6 @@
+Preférences
+===
+
+Dans cette page, vous pouvez modifier le comportement du guide des paramètres.
+
+[ ] Afficher les articles dans la définition des info-bulles (nécessite un redémarrage)
```
[the remark this answers is not in the document]
Comment (5axes) [reply resources/translations/fr_FR/about/preferences.md]: Yes I thought it was a bug :)  The workaround was  switch the page to english, activate the option , restart  Cura then switch back to our native langage. But ok I will give this information to our FB user group for thoose who want to test the new functions.

**Comment by 5axes** [comment]
Can you switch back every local link in the article with the `./` reference.   This notation works in the GitHub environement but in the Guide SettingsGuide
local

**Comment by Ghostkeeper** [comment]
> I've notice that we must use now four equal signs for article headers .

Not really a requirement. It's just that I like to keep things consistent :)

> Can we continue to review the translation ( change some texte , or add some explanations ) or do you prefer for the moment to have a break to focus your time on other contribution) .

You can continue making improvements whenever you like. For this project I specifically don't want to have all the hassle of string freezes.

> Can you switch back every local link in the article with the ./ reference. This notation works in the GitHub environement but in the Guide SettingsGuide

Oh, for me it works fine! I tested it first before making the change. How did you install the plug-in locally to test this? I can build a .curapackage file for you if you like.

File: resources/translations/fr_FR/about/preferences.md (hunk 19 above)
[the remark this answers is not in the document]
Comment (Ghostkeeper) [reply resources/translations/fr_FR/about/preferences.md]: We can also choose to translate the entire page except this text after the `[ ]`. That should work I'd expect.

**Comment by 5axes** [comment]
Yes please if you can make a cura Package it will help us to test and debug the translation.  I'm not realy sure to make the things properly concerning the test on my local pc.

**Comment by Ghostkeeper** [comment]
@5axes Here is a .curapackage (in a zip file) of the current state of master. I haven't had time yet to look at your latest changes, sorry.

[SettingsGuide2.2.99-sdk7.0.0.zip](https://github.com/Ghostkeeper/SettingsGuide/files/4372435/SettingsGuide2.2.99-sdk7.0.0.zip)

I tried installing that in Cura 4.5 and the links work fine.

**Comment by 5axes** [comment]
Unfortunatly it was not a Bug in relation with my own installation on my computer. I have the same behaviour with your Curapackage. And it's not a issue with the french translation. If I'm using the English text  or the French Text the result is the same.   with "./" no issue.
Erreur1
Errue2

**Comment by Ghostkeeper** [comment]
Is it happening with the version that's currently on the Marketplace?

It makes no sense to me. I've tested these links on many computers (Ubuntu, Windows 8, Windows 10 and MacOS Catalina) and they've been working fine. Also, the way it resolves these links is through Python's `os.path.join`, which simply concatenates the file paths to make it absolute, using the correct path separator. Like `C:\Users\Neo\AppData\Roaming\cura\4.5\plugins\SettingsGuide2\resources\articles\shell\wall_thickness.md` on Windows, or using forward slashes on MacOS and Linux.

**Comment by 5axes** [comment]
Hello,
Yes it's the same with the regular release and it was already the case with the previous Relase. I have Change recently MY pc ( Seven to W10 both in 64bits) and this issue is the same . I will ask if it's just an issue on my PC or if other French user have the same problem )   but for me using ./ is the solution to solved the problem working find in the SettingsGuide as well on the Github site.

**Comment by 5axes** [comment]
According to the feedback of french user on the FB group , yes I'm not the only one who have this issue. Something to do with the French regional settings ?

**Comment by Ghostkeeper** [comment]
I found the issue! Seems that it has to do with the backslashes in the Windows paths. I've explained the bug here: https://github.com/Ghostkeeper/SettingsGuide/commit/74c56a3f608c15fdbd2eeee043af9f51aec091de

This got introduced in version 2.2. I've tested the links on Windows with every release, but apparently I didn't test links to articles within the same folders at the last release.

Thanks for pointing out the bug. It would've taken me a long time to find it.

**Pull request merged**